spring-rabbit/src/main/java/org/springframework/amqp/rabbit/config/RabbitNamespaceUtils.java

```diff
@@ -192,7 +192,7 @@ public static BeanDefinition parseContainer(Element containerEle, ParserContext
 		String channelTransacted = containerEle.getAttribute(CHANNEL_TRANSACTED_ATTRIBUTE);
 		if (StringUtils.hasText(channelTransacted)) {
 			// Note: a placeholder will pass this test, but if it resolves to true, it will be caught during container initialization
-			if (acknowledgeMode.isAutoAck() && channelTransacted.equalsIgnoreCase("true")) {
+			if (acknowledgeMode == AcknowledgeMode.NONE && channelTransacted.equalsIgnoreCase("true")) {
```

Comment:
OK with me - we generally don't need issue numbers in test names anyway - `git blame` is our friend.